exporters/zipkin/include/opentelemetry/exporters/zipkin/zipkin_exporter.h

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include "opentelemetry/exporters/zipkin/recordable.h"
+#include "opentelemetry/ext/http/client/http_client_factory.h"
+#include "opentelemetry/ext/http/common/url_parser.h"
+#include "opentelemetry/sdk/trace/exporter.h"
+#include "opentelemetry/sdk/trace/span_data.h"
+
+#include "nlohmann/json.hpp"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace exporter
+{
+namespace zipkin
+{
+
+std::string ZIPKIN_ENDPOINT_DEFAULT = "http://localhost:9411/api/v2/spans";
```

Comment:
Make this constant, and also change the naming to `kZipkinEndpointDefault`?

Reply:
done.